src/test/TestAccount.h

```diff
@@ -163,6 +165,7 @@ class TestAccount
     }
 
     uint32_t getTrustlineFlags(Asset const& asset) const;
+    uint32_t getTrustlineFlags(TrustLineAsset const& asset) const;
```

Comment:
You should either remove this if you don't need it, or implement it if you do.

Reply:
Removed